### ISSUE TITLE
Support for Universal Links in iOS9

### ISSUE THREAD

**Issue by funge** [EddyVerbruggen/Custom-URL-scheme#111]
iOS9 no longer supports opening apps with custom URL schemes.  It instead introduces "Universal Links".

https://developer.apple.com/library/prerelease/ios/documentation/General/Conceptual/AppSearch/UniversalLinks.html

Would be great if the Custom-Url-scheme plugin could be updated to support Universal Links

**Comment by EddyVerbruggen** [comment]
Thanks for the suggestion! It would be a nice addition, perhaps it should be a new plugin, I need to investigate this a little.

Fortunately iOS 9 still supports Custom URL Schemes as it did before (see screenshots I just snagged from my iPhone 6 running iOS9 GM).

![img_5204](https://cloud.githubusercontent.com/assets/1426370/9835560/9a4191a6-59f1-11e5-97ea-c65da16bd8ba.PNG)
![img_5203](https://cloud.githubusercontent.com/assets/1426370/9835561/9a435bb2-59f1-11e5-817f-f5a575cd9f6e.PNG)

**Comment by ohh2ahh** [comment]
It would be awesome if your plugin could support both URL Schemes and Universal Links (iOS 9+) for iOS and Android ([Android supports both too](https://developers.google.com/app-indexing/android/app)).

**Comment by schoettler** [comment]
Hey Eddy!

I'm interested in building a plugin that would 'Open URLs As'. Great bump for user engagement on digest emails and social media.

Count on my support.

**Comment by dpa99c** [comment]
There's a newish plugin that handles Universal Links for both Android and iOS: https://github.com/nordnet/cordova-universal-links-plugin

I've tried it on both platforms and it works great. Note it doesn't (as of yet) handle custom URL schemes like this plugin does.

@EddyVerbruggen @nordnet Maybe you can get together and have a beautiful baby together that does both :-D

**Comment by Hirbod** [comment]
@dpa99c while a marriage between both plugins would be nice, I guess they just work too differently. Keeping them into seperated plugins should be the best case actually.

**Comment by adityasaxena** [comment]
@dpa99c Did this plugin (https://github.com/driftyco/ionic-plugin-deeplinks) end up doing both ?

**Comment by dpa99c** [comment]
@adityasaxena Seems so

**Comment by adityasaxena** [comment]
@dpa99c Of course, even that plugin owes to @EddyVerbruggen and @nordnet for inspiration.
